README.md

```diff
@@ -8,7 +8,7 @@ Example:
 GET
 ```java
 HttpRequest request = httpRequest(new TargetUrl("abv.bg")).build();
-ChitaHttp chitaClient = new ChitaHttpClient();// in most cases chitaClient should be injected
+ChitaHttp chitaClient = new HttpClient();// in most cases chitaClient should be injected
```

Comment:
Ok, when we design that we have something other in mind. Like that will take a little longer for test writing but it will be better if there are any changes in the http lib.

We will remove the interface and start writing the tests with testing servers.